acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/proxy/PassThroughAcceptanceTest.java

```diff
@@ -32,7 +33,7 @@
       Convert.toWei(TRANSFER_AMOUNT_ETHER, Convert.Unit.ETHER).toBigIntegerExact();
 
   @Test
-  public void ethGetTransactionCountReturnCorrectNumber() {
+  public void ethGetTransactionCountReturnCorrectNumber() throws IOException {
```

Comment:
Don't add `throws IOException`, deal with it at the lower level: consistency

Reply:
these were no longer necessary! removed them.